packages/next/src/server/lib/cache-handlers/default.ts

```diff
@@ -113,11 +139,18 @@ const DefaultCacheHandler: CacheHandlerV2 = {
   },
 
   async getExpiration(...tags) {
-    return Math.max(...tags.map((tag) => tagsManifest.get(tag) ?? 0))
+    const expiration = Math.max(
+      ...tags.map((tag) => tagsManifest.get(tag) ?? 0)
+    )
+
+    debug('getExpiration', { tags, expiration })
```

Comment:
don't love that we are making objects that are only used during debugging but all my ideas on eliminating this runtime cost without a build step feel overengineered.

Reply:
I guess we could consider this:

```js
const debug = process.env.NEXT_PRIVATE_DEBUG_CACHE
  ? console.debug.bind(console, 'use-cache:')
  : undefined
```

```js
debug?.('getExpiration', { tags, expiration })
```

Reply:
Yeah probably better